packages/drivers/odsp-driver/src/test/epochTests.spec.ts

```diff
@@ -55,19 +54,6 @@ describe("Tests for Epoch Tracker", () => {
 		await epochTracker.removeEntries().catch(() => {});
 	});
 
-	it("defaultCacheExpiryTimeoutMs <= maximumCacheDurationMs policy", () => {
```

Comment:
I'd keep this test and have it import the renamed thing.  It was my best effort to tie the enforced value in code to the type expressed on the IDocumentStorageServicePolicies type.

Reply:
Without this, there's no validation in CI that a larger value isn't used.  With this test, if someone tried to up that max value to 7 days or whatever, this test would fail.  Without it... eh, the UsageError thrown in ContainerRuntime's constructor would probably catch it in some e2e tests.  I still like having this here as a unit tests.

It's a super confusing way to have these policies be, I know. #techdebt